session/bootstrap.go

```diff
@@ -549,6 +550,20 @@ const (
     	key(parent_table_id, create_time),
     	key(create_time)
 	);`
+
+	// CreateGlobalTask is a table about global task.
+	CreateGlobalTask = `CREATE TABLE IF NOT EXISTS mysql.tidb_global_task (
+		id BIGINT(20) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+		type VARCHAR(256) NOT NULL,
+		dispatcher_id VARCHAR(256),
+		state VARCHAR(64) NOT NULL,
+		start_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
It is not felt necessary to set the default value to "CURRENT_TIMESTAMP"？Now the dispatcher will set the value when it is processed. If it is processed by PR, the default value will be set when it is processed by the client. Is it not as expected？